sdks/java/extensions/euphoria/euphoria-core/src/test/java/org/apache/beam/sdk/extensions/euphoria/core/testkit/junit/TestSuiteRunner.java

```diff
@@ -59,7 +59,9 @@ public TestSuiteRunner(Class<?> klass) throws Throwable {
     BeamRunnerWrapper runner =
         BeamRunnerWrapper.ofDirect().withAllowedLateness(Duration.ofHours(1));
     Class<?>[] testClasses = getAnnotatedClasses(klass);
+
     for (Class<?> testClass : testClasses) {
+      LOG.info(String.format("Testkit will schedule tests in '%s'.", testClass));
```

Comment:
Meanwhile exception messages doesnt provide good solution for formatting, logging methods do. I would prefer, String.format is uselesss here or is there some benefit?
`LOG.info("Testkit will schedule tests in {}.", testClass);`

Reply:
No benefit. Only the `info` method has unclear contract to me. I do not know how to log `Throwable` with custom message without putting `Throwable` as last formatting parameter. Something which can potentially break once SLF4J implementation changes.